PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -163,6 +174,26 @@ public class AdUnit: NSObject, DispatcherDelegate {
                 return
             }
             
+            DispatchQueue.main.async {
```

Comment:
Let's move the logic of starting the viewability tracking out from `fetchDemand`: 

- start viewability tracking on calling of activatePrebidImpressionTracker
- stop tracking on destroying the AdUnit object
- once the tracked view meets the impression tracking criteria - check the content, and if it contains PUC - track the respective impression.